packaging/ubuntu-trusty/debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+
+%:
+	dh $@
+
```

Comment:
The `debian/rules` file also needs to take care of removing the `amazon-ecs-init` binary in the `clean` rule. Something like:

   clean:
      dh $@
      rm -f amazon-ecs-init

should work. We should probably also call `go clean`, possibly by way of our `gobuild.sh` script.